prow/github/client.go

```diff
@@ -530,6 +530,17 @@ func (c *Client) CloseIssue(org, repo string, number int) error {
 	return err
 }
 
+func (c *Client) OpenIssue(org, repo string, number int) error {
```

Comment:
That makes sense. I've changed the function name to `ReopenIssue`, the command to `/reopen`, and I've added a comment.